### PR TITLE
adblock: update 1.1.12

### PR DESCRIPTION
**Pull request by dibdot**
* change wget binary naming
* change winspy source url
* documentation update

Signed-off-by: Dirk Brenken <dev@brenken.org>

### CODE REVIEW

File: net/adblock/files/adblock.conf [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 config adblock 'global'
 	option adb_enabled '1'
-	option adb_cfgver '1.2'
+	option adb_cfgver '1.3
```
Comment (hnyman) [inline net/adblock/files/adblock.conf]: A quote is missing from this line
'1.3

**Comment by dibdot** [comment]
typo fixed - thank you.

**Comment by dfc-d** [comment]
Hi,

I would actually suggest holding off on the wget change until the changes mentioned in #2729 are implemented, as, if e.g. only wget or uclient-fetch with non-working libustream-*ssl (e.g. currently the case with the default polarssl) is present the script will likely fail with less than useful messages.

Since you're require HTTPS, you have a hard dependency on SSL capability.

**Comment by dibdot** [comment]
No, this update can proceed. The adblock package itself requires the full wget package (DEPENDS:=+wget) and during runtime this will be checked again.

**Comment by dfc-d** [comment]
Here is some code for checking wget capabilties I wrote (but haven't tested yet) to use with ddns-scripts that I wrote:  Feel free to use as you see fit (since it look like for now all you are actually worried about is the SSL:
get_uclient_fetch_capabilities() {
        local basecap="forceipv ipv6"

        if [ -r "/lib/libustreamssl.so" ]; then
                local lddpath="$(which ldd)"

                [ -z "$lddpath" ] && {
                        # if we can't check ssl provider, assume broken polarssl
                        # when used by uclient-fetch (#22173)
                        echo "$basecap"
                        return
                }

                local linkedlibs="$(ldd /lib/libustream-ssl.so)"

                if [ -n "$linkedlibs" ]; then
                        # PolarSSL is an older version of libmedtls
                        # but PolarSSL ustream-ssl is currently broken
                        # when used by uclient-fetch (#22173)
                        local linkedlib="$(echo "$linkedlibs" | grep 'libmbedtls'|awk '{ print $3 }')"
                        if [ -n "$linkedlib" ]; then
                                local libpackage="$(opkg search "$linkedlib"|awk '{print $1 }')"

                                if [ "$libpackage" = "libpolarssl" ]; then
                                        # uclient-fetch with polarssl fails to do ssl
                                        echo "$basecap"
                                else
                                        # Newer mbedtls is fine
                                        echo "ssl $basecap"
                                fi
                        else
                                echo "ssl $basecap"
                        fi
                else
                        # if we can't check ssl provider, assume broken polarssl
                        # when used by uclient-fetch (#22173)
                        echo "$basecap"
                fi
        else
                echo "$basecap"
        fi
}
get_wget_capabilities() {
        local prog="wget"

        local fullpath="$(which "$prog")"

        [ -z "$fullpath" ] && {
                echo "unknown"
                return 1
        }

        local package="$(opkg search "$1"|awk '{ print $1 }')"

        [ -z "$package" ] && {
                echo "unknown"
                return 1
        }


        case "$package" in
        wget-ssl)
                echo "ssl bindip wgetproxy forceipv ipv6 extras cadir"
                ;;
        wget|wget-nossl)
                echo "bindip wgetproxy forceipv ipv6 extras"
                ;;
        uclient-fetch)
                get_uclient_fetch_capabilties || return 1
                ;;
        *)
                echo "unknown"
                return 1
                ;;
        esac
}

**Comment by dfc-d** [comment]
Oh sorry, just saw you message.  Yeah in your case there is need for all this logic.  ILTM like ddns-scripts is the only case that there is a real issue.

**Pull request merged**